### ISSUE TITLE
Web UI query for _key

### ISSUE THREAD

**Issue by sonyarianto** [arangodb/arangodb#9561]
## Steps to reproduce

1. I have key on a collection ley say 734837
2. I logged-in in Web UI and doing query using the UI from Collections menu based on _key
3. If I put 734837 it shows blank, but if I put "734837" then the data appear.

__Expected result__:
I think we do not need to add quote (such as "734837") on query based on _key field.

**Comment by dothebart** [comment]
Hi, 
please note that `_key` has to be a string by definition. AQL can differenciate between strings and numbers - thus if you want to match on `_key` you have to provide a string.

**Comment by sonyarianto** [comment]
@dothebart OK, but I think user will type on that field naturally without quote, just my opinion, it means internally the system should add quote.

**Comment by Simran-B** [comment]
@hkernbach Can you look into this?
For `_key`, `_id`, `_rev`, `_from`, `_to` at the top level of a document we can assume that they are strings.

**Comment by dothebart** [comment]
The problem here is, that the UI doesn't actually parse the query; 
`for x in knows filter x._key == @key return x`
barely matches the bind value for being there.

While the query is formaly correct its defined to have an empty result set if `@key` is an integer. Maybe the UI could visualize which datatype its choosing (null/bool/number/string) to make it more clear.

**Comment by sonyarianto** [comment]
@Simran-B nice, agree, we can assume those are strings. @dothebart thank you

**Comment by dothebart** [comment]
@Simran-B the problem is, we don't know that its compared to `_key` so while we agree, how should the webinterface know?

**Comment by Simran-B** [comment]
How do we not know that? What I meant is that the Web UI should add quote marks around the entered value if the other field is set to one of these system attributes (client-side).

But of course, we could also think about the server-side, if a number could automatically be converted to string if we know that it's a document _key for instance. Not sure if I like this though.

**Comment by Simran-B** [comment]
To clear up possible confusion: I had the filter feature of the collection browser in mind

![image](https://user-images.githubusercontent.com/7819991/63446008-bfb4fc00-c439-11e9-8de9-cfb21fa2167a.png)

In the AQL editor, it's not really possible to correct such things for queries because the web interface doesn't parse queries. This includes the bind parameter pane.

**Comment by sonyarianto** [comment]
@Simran-B Yes, I like the last image visual, I think for user it's better input 12345 rather than "12345" for _key or other fields that sould be string. Another solution is using hint for _key input should contains double quote. But I still like it naturally without any quote.

**Comment by dothebart** [comment]
Sorry for confusing this; for some reason I mentally took the wrong route in the web-UI and was mentally editing an AQL-query with bind varibles.

**Comment by dothebart** [comment]
related: https://github.com/arangodb/arangodb/issues/8166
